rskj-core/src/test/java/co/rsk/rpc/modules/eth/subscribe/EthSubscribeRequestTest.java

```diff
@@ -40,16 +40,64 @@ public void deserializeNewHeads() throws IOException {
                 new ByteArrayInputStream(message.getBytes(StandardCharsets.UTF_8))
         );
 
-        assertThat(request, instanceOf(EthSubscribeRequest.class));
-        EthSubscribeRequest subscribeRequest = (EthSubscribeRequest) request;
-        assertThat(subscribeRequest.getParams(), instanceOf(EthSubscribeNewHeadsParams.class));
+        validateParams(request, EthSubscribeNewHeadsParams.class);
     }
 
-    @Test(expected = JsonMappingException.class)
-    public void logsIsUnsupported() throws IOException {
+    @Test
+    public void deserializeLogsWithEmptyConfig() throws IOException {
```

Comment:
I think this tests belongs to commit `688f9469de848b36802c4e4d0ad3bb233ca28f11`

Reply:
I rearranged the code so `EthSubscribeLogsParams` doesn't exist until the last commit.